ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/Decision/DecisionDate.cshtml.cs

```diff
@@ -44,12 +57,11 @@ public LinkItem GetPageForBackLink(int id)
 			};
 		}
 
-		public async Task<IActionResult> OnGetAsync(int id)
+		public IActionResult OnGet(int id)
 		{
 			AdvisoryBoardDecision decision = GetDecisionFromSession(id);
 			if (decision.Decision == null) return RedirectToPage(Links.TaskList.Index.Page, new { id });
 
-			await SetDefaults(id);
```

Comment:
Why don't we need to do this anymore?

Reply:
Moved into the base class and dealt with there so it's done once, when necessary :)